build.xml

```diff
@@ -222,11 +221,10 @@
     </target>
 
     <target name="build-compiled-odd" description="Builds the compiled ODD of a specific customization submitted as absolute path with -Dcustomization.path input param." depends="canonicalize-source">
-        <basename property="odd.basename" file="${customization.path}" suffix=".xml"/>
-        <echo>${odd.basename}</echo>
+        <basename property="selectedSchema" file="${customization.path}" suffix=".xml"/>
```

Comment:
Why showing it three times?

Reply:
Because 1) you can run the target separately, and 2) it provides some orientation when running the full transformation scenario with `ant dist`. We don't have a lot control about the output generated by the TEI stylesheets, so this "echo" serves as a kind of intermediary heading.